app/controllers/admin/visit_requests_controller.rb

```diff
@@ -15,7 +15,7 @@ def event
     end
 
     def visit_requests
-      @visit_requests ||= search_against(base_scope).includes(:user, :event).order(:id)
```

Comment:
Can you add tit as a separate PR too?
As a part of - https://github.com/pivorakmeetup/pivorak-web-app/issues/379